robottelo/ui/computeresource.py

```diff
@@ -38,6 +38,27 @@ def _clean_value(self, name, value):
                 value = [value]
         return value
 
+    def _assign_locator_value(self, target, value):
+        """Assign provided value to page element depending on the type of that
+        element
+        """
+        if (self.page.element_type(target) == 'span' or
+                self.page.element_type(target) == 'select') and ' (' in value:
```

Comment:
This causes a issue as , it will try to check it for check-box with a boolean value and throw a typeerror.
So first, trying to check the element type and then the value